spec/external_item_with_optional_org_spec.rb

```diff
@@ -0,0 +1,25 @@
+include RailsMultitenant
+
+describe ExternalItemWithOptionalOrg do
+
+  let!(:external_item_without_org) { as_external_org(nil) { described_class.create! } }
+
+  let!(:external_item_with_org) { as_external_org(1) { described_class.create! } }
+  let!(:external_item_with_other_org) { as_external_org(2) { described_class.create! } }
```

Comment:
This ensures that we're not leaking models with different organization ids to other organizations. We don't really care about testing organization 2 as long as this model doesn't show up in the nil organization or organization 1.